app/src/pages/inside/common/accordion/accordion.jsx

```diff
@@ -31,7 +31,10 @@ export const Accordion = ({ tabs, headerClassNames, contentClassNames, toggleTab
             key={tab.id}
             tab={tab}
             onClick={() => toggleTab(tab.id)}
-            headerClassNames={tab.shouldShow ? headerClassNames : 'hidden'}
+            headerClassNames={cx(headerClassNames, {
```

Comment:
Are you sure that `headerClassNames` and `contentClassNames` pass as props? I do not see their transfers in the code.

Reply:
This is an optional props, but if it is, then it is passed like this.

Reply:
But I don't see that we are passing these parameters at all. Do we really need them?

Reply:
Yes, they are not needed yet, I will delete them